src/OpenTelemetry.Instrumentation.Owin/OwinInstrumentationOptions.cs

```diff
@@ -17,18 +18,20 @@ public class OwinInstrumentationOptions
     /// Initializes a new instance of the <see cref="OwinInstrumentationOptions"/> class.
     /// </summary>
     public OwinInstrumentationOptions()
+        : this(new ConfigurationBuilder().AddEnvironmentVariables().Build())
```

Comment:
"Wouldn't it be the best option to take `IConfiguration` from the user instead of building it with environment variables ourselves? Is the intention here to avoid a public API change?

Reply:
Ya this is just about backwards compatibility and not breaking the public API. Nothing should be using this ctor but there is a chance some user is calling it. Perhaps in a test. But this is a prerelease package so we _could_ break it and remove this ctor leaving just have the `internal` one. LMK what you think.

Reply:
I'm fine with this approach.